core/store.go

```diff
@@ -6,23 +6,16 @@ import (
 	"strings"
 	"time"
 
+	"encoding/json"
```

Comment:
The import grouping here is off - `encoding/json` should be a part of the system imports group, e.g. "strings", "time", etc.